getting_started/workflow/best_practices/autoloads_versus_internal_nodes.rst

```diff
@@ -1,103 +1,95 @@
-individual scene or node subtree.
-
-If X is data, then the solution is either 1) to create a new type of
-:ref:`Resource <class_Resource>` to share the data, or 2) to store the data
-in an object to which each node has access (nodes within a scene can use
-`get_owner()` to fetch the scene root for example).
-
-So when *should* one use an autoload?
-
-- **Static Data**: if you need static data, i.e. data that should be
-  associated with a class (so there is only ever one copy of the data), then
-  autoloads are good opportunities for that. Static data doesn't exist in
-  Godot's scripting API, so autoload singletons are the next best thing. If
-  one creates a class as an autoload, and never creates another copy of that
-  class within a scene, then it will function in place of a formal singleton
-  API.
-
-- **Convenience**: autoloaded nodes have a global variable for their name
-  generated in GDScript. This can be very convenient for defining objects
-  that should always exist, but which need object instance information.
-  The alternative is to create a namespace script: a script that's purpose
-  is only to load and create constants to access other Script or PackedScene
-  resources, resulting in something like ``MyAutoload.MyNode.new()``.
-
-  - Note that the introduction of script classes in Godot 3.1 questions
-    the validity of this reason. With them, one can access scripts using an
-    explicit name from GDScript. Using an autoload to get to a namespace
-    script becomes unnecessary, e.g. ``MyScriptClass.MyPreloadedScript.new()``.
-
-If the singleton is managing its own information and not invading the data of
-other objects, then it's a great way to create a "system" class that handles
-a broad-scoped task. For example a targeting system, quest system, or dialogue
-system would be great use cases of singleton implementations.
+Auto-loaded nodes can simplify your code in some cases:
+
+- **Static Data**: if you need data that is exclusive to one class, like a
+  database, then an autoload can be a good tool. There is no scripting API in
+  Godot to create and manage static data otherwise.
+
+- **Systems with a wide scope**: If the singleton is managing its own
+  information and not invading the data of other objects, then it's a great way to
+  create systems that handle broad-scoped tasks. For example, a quest or a
+  dialogue system.
+
+Until Godot 3.1, another use was just for convenience: autoloads have a global
+variable for their name generated in GDScript, allowing you to call them from
+any script file in your project. But now, you can use the ``class_name`` keyword
+instead to get autocompletion for a type in your entire project.
+
+.. note::
+
+   Autoload is not exactly a Singleton. Nothing prevents you from instancing
+   copies of an auto-loaded node. It is only a tool that makes a node load
+   automatically at the root of your scene tree, regardless of your game's node
```

Comment:
Correction: "that makes a node load automatically \[as a child of\] the root of your scene tree..." I would recommend explicitly telling people that it will exist at `/root/<autoload_name>`. Clarity is key here.